quickfixj-core/src/test/java/quickfix/SessionResetTest.java

```diff
@@ -137,6 +137,16 @@ public void onMissedHeartBeat() {
 
         public void onHeartBeatTimeout() {
         }
+
+        public void onResendRequestSent(int beginSeqNo, int endSeqNo, int currentEndSeqNo) {
+        }
+
+        public void onSequenceResetReceived(int newSeqNo, boolean gapFillFlag) {
+        }
+
+        @Override
```

Comment:
nitpick: either remove `@Override` or put it to all other methods as well.

Reply:
Fixed